util/jsonresponse.py

```diff
@@ -130,14 +130,18 @@ def _deployment_particles(self, ds, stream_key, parameters, external_includes):
             if lat_data is not None and lon_data is not None:
                 data['lat'] = lat_data
                 data['lon'] = lon_data
-                params.extend(('lat', 'lon'))
+        # always include lat/lon
+        params.extend(('lat', 'lon'))
```

Comment:
Is it correct to include lat/lon when we do note have values for them?

Reply:
lines 127 - 132 apply to gliders only and essentially rename GPS based lat/lon as simply lat or lon. Some non-glider data does have lat/lon, e.g. GI01SUMO_RID16_02_FLORTD000, which were not being reported. At this point, we are actually saying we want to include lat/lon; if they are not available, they are reported as missing in lines 161 - 163 and removed from the requested param list in 165.

I think it is more efficient to do this than to attempt to determine ahead of time if lat/lon are available.